setup.py

```diff
@@ -27,7 +26,9 @@
 
     packages=['gdal2mbtiles'],
     include_package_data=True,
-    install_requires=['future', 'numexpr', 'numpy', 'pyvips', 'webcolors'],
+    install_requires=[
+        'future', 'numexpr', 'numpy', 'pyvips==2.1.8', 'webcolors'
```

Comment:
It's not recommended to pin packages in a library's setup.py (https://packaging.python.org/discussions/install-requires-vs-requirements/) - might be worth doing `pyvips<=2.1.8` instead?